PYME/ui/traitsui_constants.py

```diff
@@ -20,6 +20,7 @@
 
 #: Define platform and wx version constants:
 is_mac = sys.platform == "darwin"
+is_dark = wx.SystemSettings.GetAppearance().IsDark()
```

Comment:
We need to wrap this in a `try ... except AttributeError` block to maintain backwards compatibility with older versions of wx (necessary for now for package resolvability / installability)